csharp/download_deps.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+
+set -e -x
+
+cd "$(dirname "$0")"
+
+BUILD_DIR="$(pwd)"
+DOWNLOAD_DIR="${BUILD_DIR}/deps"
+SDK_VERSION="13.4.0-b6699-4526b-WORKER-SNAPSHOT"
+mkdir -p "${DOWNLOAD_DIR}"
+
+# Get the tools
+# * Core SDK for all platforms to enable building workers for MacOS, Windows or Linux
```

Comment:
If this refers to the Worker SDK, could we change "Core" to "Worker" for clarity?